mp/src/materialsystem/stdshaders/buildshaders_inc.bat

```diff
@@ -141,10 +138,9 @@ nmake /S /C -f makefile.%inputbase%
 REM ****************
 REM Copy the inc files to their target
 REM ****************
-
 if exist "inclist.txt" (
- 	echo Publishing shader inc files to target...
- 	perl %SrcDirBase%\devtools\bin\copyshaderincfiles.pl inclist.txt
+	echo Publishing shader inc files to target...
+	perl %SrcDirBase%\devtools\bin\copyshaderincfiles.pl inclist.txt
```

Comment:
I'd like to remove the dependency on Valve's old ShaderCompile. I know this involves some shader recompiling but we really only use `SDK_LightmappedGeneric` and `PBR`, for what it's worth. Makes no sense that we'd want to go 95% of the way there.

Reply:
Further discussion brought to light that it requires recompiling *all* shaders, including standard ones, so that the new header works with them and the `.inc` files are appropriate. This can be tackled later, I'll accept the hybrid solution for now.